content/swift/concepts/protocols/terms/encodable/encodable.md

```diff
@@ -1,20 +1,28 @@
 ---
-Title: 'Encodable'
-Description: 'Enables object data to be encoded for use with Application Programming Interfaces.'
+Title: 'Encodeable'
+Description: 
+ - 'A type that can convert itself into an external representation.'
+ - 'handles data encoding tasks'
 Subjects:
-  - 'iOS'
-  - 'Mobile Development'
+  - 'iOS 8.0+ | iPadOS 8.0+ | macOS 10.10+ | Mac Catalyst 13.0+ | tvOS 9.0+ | watchOS 2.0+ | visionOS 1.0 + Beta'
 Tags:
   - 'Protocols'
   - 'APIs'
+  - 'encoding'
 CatalogContent:
   - 'learn-swift'
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-**`Encodable`** is a protocol that enables a type's data to be encoded for use with Application Programming Interfaces ([APIs](https://www.codecademy.com/resources/docs/general/api)).
+**`Encodable`** is a protocol that enables a type's data to be encoded and decoded. 
+
+> **Note:** A type that can encode itself to an external representation. The protocol is related to the following protocols
+- Codeable
+- Decodeable
+- CodingKey
+- CodingKeyRepresentable
+- CodingUserInfoKey
 
-> **Note:** If data also needs to be decoded, it's convenient to use the `Codable` protocol because it includes both the `Encodable` and `Decodable` protocols.
```

Comment:
I'd like you not to delete this line.

Reply:
I added the line again in the document.

Reply:
@BBrandl948 It's _best left as a note_ to differentiate its content from the rest of the introduction, since it's something the reader should be aware of outside the term